flavour/azure/flavourinit.sh

```diff
@@ -4,6 +4,12 @@ echo "Logging in..."
 tenantArg=()
 
 if [ "X${AZ_TENANTID}X" != "XX" ]
+then
+  echo "Warning: This configuration uses the DEPRECATED AZ_TENANTID environment variable. Please use ARM_TENANT_ID instead."
```

Comment:
Do you maybe want to add a hint when this will be removed, for example with version 6.0 or 7.0 ?

Reply:
That's a good idea, but I'm not really sure when. 😄 Would "removed in a later version" be sufficient as well? We don't really have  release management for CloudControl as of right now

Reply:
'removed in a later version' would be nice, sure.
How about first major release after Q3-2024? This way you could bind it to a major change AND ensure a certain time for the users to adjust.

Reply:
Makes sense.